app/services/session_encryptor.rb

```diff
@@ -1,4 +1,6 @@
 class SessionEncryptor
+  MARSHAL_SIGNATURE ||= 'BAh'.freeze
```

Comment:
See https://github.com/18F/identity-idp/pull/1358/commits/8a427ca6f3345ce22c0e87dfd1b837debeac88bc for why this is memoized.